src/main/java/core/basesyntax/factory/BetDaoFactory.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.factory;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+
+public class BetDaoFactory {
+    private static BetDao INSTANCE;
```

Comment:
```suggestion
    private static BetDao instance;
```
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names